### ISSUE TITLE
All Items dissapeared after being semi-invisible

### ISSUE THREAD

**Issue by AssassinBear23** [CyclopsMC/ColossalChests#156]
Playing the new 1.18.2 Vault Hunters mod pack on a low grade server (VPS).

After moving everything from chests to the colossal chest everything went fine, I put it directly into the chest (4^3), after I upgraded the size and put everything in again (broke the core so everything yeeted out). Once again no problem, put it in directly into the chest.

Now I have a interface block to which a ME storage bus is connected which is part of a simple 1 ME Drive system where all the drives are partitioned. Friend of mine wanted to put all items in the colossal chest in through the ME terminal so that anything that can go onto the disks does, fair enough.

We do that, and then when we look in the chest, theres no items, but it does show up in the ME Terminal, therefore the semi-invisible. After this there have been no server restarts (still done by hand). When we came back all the items that were meant to be in the colossal chest weren't there, in the chest nor in the ME system. 

Is the CurseForge version also needed?

Minecraft Version: 1.18.2
AE2: Version 11.1.4
Colossal: Version 1.1.0

**Comment by rubensworks** [comment]
Did this occur just once, or can you consistently reproduce this as you described?

Also, can you retry without any other mods installed?

**Comment by AssassinBear23** [comment]
> Did this occur just once, or can you consistently reproduce this as you described?

I will try it again, though this would not be on the server due to it influencing other people.

> Also, can you retry without any other mods installed?

I will try later today

**Comment by AssassinBear23** [comment]
So I tried it again on the server, and on a singleplayer creative world (Both with the modpack: Vault Hunters 1.18.2 Third Edition), and the results are different.

Singeplayer: Everything worked fine. Was smooth, and no visual bug either (Will make another bug report about that, or reopen one if it already exists)

Multiplayer: Same thing happened, semi-invisible items, but it was fixed after replacing the ME Storage Bus. Attached a imgur link for the video where I try and find this out.

[Imgur Link](https://imgur.com/a/sfMTRA1).

I will try something in SP and update this comment if I find something.

Edit: So I found where the problem lies, and it seems to be more of a AE2 issue. I've added a youtube link showcasing what the bug actually is. 
[Youtube link](https://youtu.be/dJ3CVmpLNm0)


In short, it comes down to when you break a part of the colossal chest, it becomes just normal blocks and not an inventory anymore. So the ME Bus doesn't work, when you place the block back and make the colossal chest whole again, the storage bus seems to see it as another inventory, and not the same one as before (assuming you don't break the Chest Core, because then it does indeed become a 'different' inventory. So the fix would then be to remove and place the ME Storage Bus again. Then it correctly shows the inventory and moves the items in again.

**Comment by rubensworks** [comment]
Thanks for the info. Looks like this is an AE2 bug then (since Colossal Chest is properly invalidating capabilities upon changes).
